### ISSUE TITLE
Support for Cursor IDE (VS Codium)

### ISSUE THREAD

**Issue by sam-goodwin** [coder/coder#13267]
I'd love to use Coder with Cursor instead of vanilla VS Code. 

I think Cursor is built on https://vscodium.com/

**Comment by sam-goodwin** [comment]
As far as I can tell, there's no way to install the extension in Cursor. Unless I'm totally missing something.

**Comment by matifali** [comment]
Will puvlishung the coder remote extension to [Open VSX registry](https://open-vsx.org/) solves this problem?

**Comment by kylecarbs** [comment]
It should!

**Comment by matifali** [comment]
coder.remote extension depends on MS remote extension that's also nit available in the OpenVSX registry.

**Comment by sam-goodwin** [comment]
I see that Coder is in the marketplace but am getting the following error:
> Cannot activate the 'Coder' extension because it depends on an unknown 'ms-vscode-remote.remote-ssh' extension.

**Comment by sam-goodwin** [comment]
Relevant discussion: https://github.com/VSCodium/vscodium/discussions/693

I think coder needs to be updated to use: https://open-vsx.org/extension/jeanp413/open-remote-ssh

**Comment by kylecarbs** [comment]
I guess we need to adjust our build to create two releases? One that depends on VS Code Remote SSH and another that depends on Open Remote SSH?

**Comment by matifali** [comment]
@kylecarbs Can we fallback on one-remote-ssh?

**Comment by bpmct** [comment]
The Coder extension works fine with Cursor out of the box. We'll make a module/guide for it soon.

**Comment by matifali** [comment]
Yes, recent testing shows that it works, and our [vscode desktop module](https://registry.coder.com/modules/vscode-desktop) can work as if we just replace `vscode` with `cursor` in URI.

Question: Should we create a new module or add support for the cursor in the vscode-desktop module?

**Comment by matifali** [comment]
Closed in https://github.com/coder/modules/pull/290

**Comment by matifali** [comment]
Available at 
https://registry.coder.com/modules/cursor
